proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,315 @@
+# AWS CLI Contributing Guide
+
+Proposal         | Metadata
+---------------- | -------------
+**Author**       | Kenneth Daily
+**Status**       | Proposed
+**Created**      | 19-October-2021
+
+## Abstract
+
+This document proposes standards and processes to improve and publish community
+expectations for contributions. It defines the process for proposing,
+submitting, and reviewing a contribution. This proposal will make it easier for
+the community to successfully contribute to the AWS CLI.
+
+## Motivation
+
+As of 2021-11-02, there are 189 pull requests and 465 issues in the AWS CLI
+GitHub repository. 52 pull requests and 81 issues have not been updated in more
+than a year. This volume indicates that there are motivated users who want to
+contribute to the AWS CLI but are unable to do so. Further, it demonstrates that
+there are users who have ideas for features but do not see them incorporated.
+From the community perspective, the volume of open issues and pull requests
+indicates a lack of awareness or communication with the community to respond to
+their needs.
+
+### Goals
+
+The approach for customer contributions should satisfy the following goals:
+
+1. Users should know how they can help. It should be straightforward for
+   community members to make high quality contributions.
+1. No requests should exist without a response. Communication from maintainers
+   should happen in a timely manner, even (especially!) if that’s saying no. An
+   initial response to open PRs should occur within [TBD] business days.
+1. A contribution should be completed through the efforts of the contributor and
+   maintainers and not be left abandoned. The maintainers are responsible see it
+   through to a conclusion.
+1. We should prioritize the things that people want based on feedback they
+   provide on issues. Users should know how to provide that feedback.
+1. Users should know exactly what to expect when interacting with the
+   maintainers. The next steps should be clearly defined.
+
+### Examples
+
+#### Limited information on where and what to contribute
+
+The AWS CLI currently has code, documentation, and examples published publicly
+on a number of GitHub repositories. We have hundreds of open issues and pull
+requests across these areas. However, we do not have a guide for users to know
+what things are available for them to contribute on or what the process or
```

Comment:
Suggest shortening to "where they can contribute"

Reply:
done